entities/storobj/storage_object.go

```diff
@@ -234,19 +234,28 @@ func FromBinaryOptional(data []byte,
 		}
 	}
 
```

Comment:
There is other multivector code we could bring over from 1.29, but I don't know if we need to bring it all over or just this minimal amount?

Run this from the root of the weaviate repo to see diff:
```sh
git diff main stable/v1.28 -- entities/storobj/storage_object.go
```

Also, another thing we could do is instead of loading up all the multivectors like this, we could instead just silently discard them on `1.28+`, which would avoid any confusion about if they are/not supported.